### PR TITLE
Less online banner

### PR DESCRIPTION
**Pull request by Discordius**
Adds banner to LessWrong frontpage for LessOnline

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206937055038871) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/common/HomeLatestPosts.tsx [hunk 1]
```diff
@@ -165,6 +165,7 @@ const HomeLatestPosts = ({classes}: {classes: ClassesType}) => {
                 filterSettingsToggleLabels.desktopHidden}
               showIcon={false}
               onClick={changeShowTagFilterSettingsDesktop}
+              color={isLW ? "white" : undefined}
```
Comment (b0b3rt) [inline packages/lesswrong/components/common/HomeLatestPosts.tsx:168]: Possibly the unit tests are complaining about this?

Comment (b0b3rt) [reply packages/lesswrong/components/common/HomeLatestPosts.tsx:168]: oh nvm this isn't an inline style

File: packages/lesswrong/components/Layout.tsx [hunk 2]
```diff
@@ -143,9 +143,58 @@ const styles = (theme: ThemeType): JssStyles => ({
...
+      fontSize: '20px',
+      margin: 0,
+      lineHeight: '1.2',
+      marginBottom: 8
+    },
+    '& button': {
+      ...theme.typography.commentStyle,
+      backgroundColor: theme.palette.panelBackground.translucent3,
+      border: 'none',
+      color: 'black',
```
Comment (b0b3rt) [inline packages/lesswrong/components/Layout.tsx:179]: missed this one

File: packages/lesswrong/components/Layout.tsx [hunk 3]
```diff
@@ -143,9 +143,58 @@ const styles = (theme: ThemeType): JssStyles => ({
...
+      textAlign: 'center',
+      padding: 8
+
+      // color: #000;
+      // border: 0;
+      // text-align: center;
+      // border-radius: 3px;
+      // background-color: rgba(255, 255, 255, .6);
+      // padding: 8px;
+      // font-family: 'Gill Sans';
```
Comment (b0b3rt) [inline packages/lesswrong/components/Layout.tsx:190]: cleanup

File: packages/lesswrong/components/icons/SettingsButton.tsx [hunk 4]
```diff
@@ -27,6 +27,11 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
   blackLabel: {
     color: theme.palette.text.primary,
+  },
+  whiteLabel: {
+    [theme.breakpoints.up('lg')]: {
+      color: theme.palette.text.alwaysWhite
+    }
```
Comment (oetherington) [inline packages/lesswrong/components/icons/SettingsButton.tsx:34]: White is the default so this has unexpected consequences such as changing the colour of the "sorted by" button in the dialogue to add a draft post to a sequence (there are probably other places too, but this is the first one I found):

Screenshot 2024-03-27 at 11 18 06

This also means all the places where you're explicitely adding `color="white"` or `color={isLW ? "white" : undefined}` aren't actually doing anything as they're already using the white color anyway.

Comment (Discordius) [reply packages/lesswrong/components/icons/SettingsButton.tsx:34]: Oh, it's the default, oops. Will fix.

Comment (Discordius) [reply packages/lesswrong/components/icons/SettingsButton.tsx:34]: (Fixed)

Comment (oetherington) [reply packages/lesswrong/components/icons/SettingsButton.tsx:34]: This is still broken, but just in a different way:
Screenshot 2024-03-27 at 23 16 12

The correct way to fix this is probably to add a third "grey" variant that doesn't apply either the "black" or "white" classes but just uses the current `grey[600]` value, and this should be the default (as it currently is).

**Pull request merged**